app/models/vedastore/ballot_style.rb

```diff
@@ -0,0 +1,13 @@
+class Vedastore::BallotStyle < ActiveRecord::Base
+  include XsdRailsFunctions
+  include Vedaspace::BallotStyle
+  
+  belongs_to :election
+  has_one :external_identifier_collection, :as=>:identifiable
+  has_many :ordered_contests
```

Comment:
So there's a little bit of that in place. In particular "belongs to". In VEDaSpace it doesn't actually do anything. But in VEDaStore there's an extra bit of code that hooks into VEDaSpace definitions to set up the rails belongs_to association. I'm not sure if it makes sense to set them all up that way (theoretically the general relationships are already defined in VEDaSpace, and these are actually specific to how the storage system implements the relationships). I think it needs more thought and we should create an issue for it for a future enhancement?